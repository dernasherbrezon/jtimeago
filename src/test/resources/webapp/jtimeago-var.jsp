<%@page contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" trimDirectiveWhitespaces="true" %>
<%@ taglib prefix="ta" uri="https://github.com/dernasherbrezon/jtimeago" %>
<%
	java.util.Date comment = new java.util.Date(1534320716000L);
	pageContext.setAttribute("comment", comment);
%>
<ta:formatDate value="${comment}" var="outputCommentDate" pattern="dd MMM yyyy HH:mm" />
${outputCommentDate}