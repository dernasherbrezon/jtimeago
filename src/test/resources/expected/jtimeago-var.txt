15 Aug 2018 08:11